e2e/process-services-cloud/start-task-custom-app-cloud.e2e.ts

```diff
@@ -96,7 +96,36 @@ describe('Start Task', () => {
         startTask.addDueDate('invalid date')
                  .blur(startTask.dueDate)
                  .validateDate(dateValidationError)
-                 .checkStartButtonIsDisabled();
+                 .checkStartButtonIsDisabled()
+                 .clickCancelButton();
+    });
+
+    it('[C290182] Should be possible to assign the task to another user', () => {
+        tasksCloudDemoPage.openNewTaskForm();
+        startTask.addName(standaloneTaskName)
+                 .addAssignee('Super Admin')
+                 .clickStartButton();
+        tasksCloudDemoPage.myTasksFilter().clickTaskFilter();
+        expect(tasksCloudDemoPage.getActiveFilterName()).toBe('My Tasks');
+        tasksCloudDemoPage.taskListCloudComponent().getDataTable().checkContentIsNotDisplayed(standaloneTaskName);
+    });
+
+    it('[C291953] Assignee field should display the logged user as default', () => {
+        tasksCloudDemoPage.openNewTaskForm();
+        expect(startTask.getAssignee()).toContain('Admin', 'does not contain Admin');
+        startTask.clickCancelButton();
+    });
+
+    it('[C291956] Should be able to create a new standalone task without assignee', () => {
+        tasksCloudDemoPage.openNewTaskForm();
+        startTask.addName(standaloneTaskName);
```

Comment:
I would suggest to use a different name then the one used above(standaloneTaskName)